schema.json

```diff
@@ -1,236 +1,293 @@
 {
-    "$schema": "http://json-schema.org/draft-07/schema",
-    "type": "object",
-    "required": ["notificationList"],
-    "additionalProperties": false,
-    "properties": {
-        "$schema": {
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Generated schema for notifications/menu",
+  "type": "object",
+  "properties": {
+    "$schema": {
+      "type": "string"
+    },
+    "notificationList": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "date": {
             "type": "string"
-        },
-        "notificationList": {
+          },
+          "title": {
+            "type": "string"
+          },
+          "updateMode": {
+            "type": "string"
+          },
+          "comment": {
+            "type": "string"
+          },
+          "prefixItemList": {
             "type": "array",
             "items": {
-                "type": "object",
-                "required": ["name", "description"],
-                "additionalProperties": false,
-                "properties": {
-                    "date": {
-                        "type": "string"
+              "type": "object",
+              "properties": {
+                "$type": {
+                  "type": "string"
+                },
+                "value": {
+                  "type": "string"
+                },
+                "label": {
+                  "type": "string"
+                },
+                "url": {
+                  "type": "string"
+                },
+                "indent": {
+                  "type": "number"
+                },
+                "color": {
+                  "type": "string"
+                }
+              },
+              "required": [
+                "$type",
+                "label"
+              ]
+            }
+          },
+          "metLists": {
+            "type": "object",
+            "properties": {
+              "itemList": {
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "$type": {
+                      "type": "string"
                     },
-                    "name": {
-                        "type": "string"
+                    "label": {
+                      "type": "string"
                     },
-                    "description": {
-                        "type": "string"
+                    "value": {
+                      "type": "string"
                     },
-                    "updateChannel": {
-                        "type": "string",
-                        "enum": ["stable", "testing", "unstable"]
+                    "url": {
+                      "type": "string"
                     },
-                    "prefixItemList": {
-                        "type": "array",
-                        "items": {
-                            "$ref": "#/definitions/item"
```

Comment:
This schema looks a lot less strict than mine; perhaps we can address that in a later update? For example, these items share a common definition which uses `oneOf` to ensure each type of item has the right properties.